pkg/kubelet/image_manager_test.go

```diff
@@ -107,6 +118,10 @@ func TestDetectImagesInitialDetect(t *testing.T) {
 	require.True(t, ok)
 	assert.Equal(zero, withContainer.firstDetected)
 	assert.True(withContainer.lastUsed.After(startTime))
+	withContainerUsingNoNameImage, ok := manager.getImageRecord(imageName(2))
```

Comment:
If I understand correctly, `imageName(1)` corresponds to `withContainerUsingNoNameImage`.